yt/frontends/enzo/answer_testing_support.py

```diff
@@ -104,7 +104,7 @@ def __call__(self):
             field = ad[k].d
             for xmin, xmax in zip(self.left_edges, self.right_edges):
                 mask = (position >= xmin) * (position <= xmax)
-                exact_field = np.interp(position[mask], exact["pos"], exact[k])
+                exact_field = np.interp(position[mask].value, exact["pos"], exact[k])
```

Comment:
note that `.value` returns a copy of the underlying data, which is wasteful here. Using a view instead will reduce memory usage
```suggestion
                exact_field = np.interp(position[mask].ndview, exact["pos"], exact[k])
```